_includes/header.html

```diff
@@ -4,15 +4,16 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-
+    <meta name="test" content="testing" />
```

Comment:
Is this a work in-progress?

Reply:
@amitamrutiya, do you think that you can assist @rishabhsharma1997 in finishing off this work?

Reply:
Sure 💯